crates/ruma-signatures/src/keys.rs

```diff
@@ -90,7 +87,31 @@ impl Ed25519KeyPair {
     /// generated from the private key. This is a fallback and extra validation against
     /// corruption or
```

Comment:
I know you didn't do this but any idea why the comment just stops?

Reply:
Nope, sorry